drpg/sync.py

```diff
@@ -126,7 +127,10 @@ def _file_path(self, product: Product, item: DownloadItem) -> Path:
         )
         product_name = _normalize_path_part(product["products_name"], self._compatibility_mode)
         item_name = _normalize_path_part(item["filename"], self._compatibility_mode)
-        return self._library_path / publishers_name / product_name / item_name
+        if  not self._omit_publisher:
```

Comment:
Please fix formatting so the linter does not complain
```suggestion
        if not self._omit_publisher:
```